app/build.gradle

```diff
@@ -240,7 +239,6 @@ android {
         }
         debug {
             minifyEnabled false
-            testCoverageEnabled true
```

Comment:
I think we should keep test coverage, though?

Reply:
`SettingsActivityUnitTests` failures does not seem to happen on main, somehow (it fails with other errors though): https://github.com/commons-app/apps-android-commons/actions/runs/6583628746/job/17886967295

Reply:
Fixed